zerver/webhooks/codebase/view.py

```diff
@@ -0,0 +1,83 @@
+import json
+from typing import Any, Dict, Iterable
+
+from django.http import HttpRequest, HttpResponse
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message, \
+    UnexpectedWebhookEventType
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+def get_event_type(payload: Dict[str, Any]):
+    return (payload['type'])
+
+def get_topic_type(payload: Dict[str, Any]):
+
+    if payload['type'] == "ticket_update":
+        payload = payload['payload']
+        payload = json.loads(payload)
+
+        return payload['ticket']['summary']
+    else:
+        payload = payload['payload']
+        payload = json.loads(payload)
+
+        return payload['summary']
+
+def ticket_creation(payload: Dict[str, Any], user_profile: UserProfile):
+
+    payload = payload['payload']
+    payload = json.loads(payload)
+
+    type = payload['type']['name']
+    name = payload['assignee']['name']
+    category = payload['category']
+    url = payload['url']
+
+    body = "A ticket of **[{}]({})** type and category **{}** has been created by **{}**".format(type, url, category, name)
+
+    return body
+
+def ticket_update(payload: Dict[str, Any], user_profile: UserProfile):
+    payload = payload['payload']
+    payload = json.loads(payload)
+
+    id = payload['ticket']['id']
+    type = payload['ticket']['category']
```

Comment:
Both `id` and `type` are reserved keywords in Python, it is a good habit to use descriptive alternatives such as `ticket_type` or `ticket_id` instead!